src/cunumeric/mapper.cc

```diff
@@ -98,8 +98,8 @@ std::vector<StoreMapping> CuNumericMapper::store_mappings(
       auto& inputs  = task.inputs();
       auto& outputs = task.outputs();
       mappings.push_back(StoreMapping::default_mapping(inputs[0], options.front()));
-      mappings.back().policy.exact = true;
```

Comment:
I assume this change was related to this comment

> removing the exact mapping constraint for FFT inputs (which wasn't doing exactly what it was meant to do)

Can you expand on that? What was wrong with the way it was done before?

Reply:
The dimension ordering doesn't take the store transforms into account and is always absolute with respect to the original coordinate space. Because of this, the `exact` constraint alone cannot guarantee the exact instance in C order that cuFFT expects, if the store was transposed. Currently, the only way to check this is to make an affine accessor with the affine transform and call `is_dense_row_major`. Once we start supporting relative dimension ordering, I guess we could put back the exact constraint and revert the change in the FFT task, but even then the exact constraint is unnecessary for certain types of FFTs we are doing; for example, forward and backward FFTs over axes make copies of the input internally, so the exact constraint only makes a copy that would be thrown away immediately: https://github.com/nv-legate/cunumeric/blob/branch-22.05/src/cunumeric/fft/fft.cu#L262

Reply:
FYI, the exact constraint we would potentially want is something like this:
```
 // scalars[2] being true means we're doing FFT over axes, which makes a copy inside the task
if (!task.scalars()[2].value<bool>()) mappings.back().policy.exact = true;
```